src/main/java/org/jabref/gui/menus/FileHistoryMenu.java

```diff
@@ -53,6 +53,7 @@ private void setItems() {
     private void addItem(Path file, int num) {
         String number = Integer.toString(num);
         MenuItem item = new MenuItem(number + ". " + file);
+        item.setMnemonicParsing(false);
```

Comment:
Could you please add a short comment explaining why mnemonic parsing is disabled for files, so that our future us doesn't remove it in a few months because the original reason for its introduction has been forgotten.

Reply:
Done